webapp/channels/src/components/drafts/drafts_link/drafts_link.tsx

```diff
@@ -66,9 +104,23 @@ function DraftsLink() {
                             />
                         </span>
                     </div>
-                    {count > 0 && (
-                        <ChannelMentionBadge unreadMentions={count}/>
-                    )}
+                    {
+                        draftCount > 0 &&
+                        <ChannelMentionBadge
+                            unreadMentions={draftCount}
+                            icon={pencilIcon}
+                        />
+                    }
+
+                    {
+                        teamScheduledPostCount > 0 &&
+                        <ChannelMentionBadge
+                            unreadMentions={teamScheduledPostCount}
+                            icon={scheduleIcon}
+                            className={`scheduledPostBadge ${scheduledPostsHasError ? 'persistent' : ''}`}
```

Comment:
You can use classnames for this:
```suggestion
                            className={classnames('scheduledPostBadge', {persistent: scheduledPostsHasError})}
```

Reply:
Done.